_includes/comparison.html

```diff
@@ -360,6 +354,31 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>yes</td>
       </tr>
     </tbody>
+    <thead style="background-color: white; color: #ff4d67">
+      <tr>
+        <th scope="col">Advantages & Drawbacks</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th scope="row">Advantages</th>
+        <td>Istio can be adapted and extended like no other Mesh. Thanks to it's big and vivid community, it's constantly improved. Istio is <a href="https://istio.io/about/community/customers/" target="_blank">used in production.</a></td>
```

Comment:
Do you feel that its support for the different platforms is worth mentioning?

Reply:
Absolutely, thanks